### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by juliatan**
Following the retro on 17 June, here are some proposed changes to the PR template.

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -13,13 +15,15 @@ _Description of the feature or fix with link to the issue if existing_
...
 
-- [ ] I have updated mockServer
-- [ ] I've added analytics as relevant
-- [ ] I have run `npm test`
-- [ ] I have run `npm run test:i18n`
+- [ ] Analytics/tracking events have been added (if needed)
+
+## Checklist (for reviewers)
+
+- [ ] Checked against Figma designs
```
Comment (aikewoody) [inline docs/pull_request_template.md:24]: How about also adding `(if relevant)` to this line?

**Review by aikewoody — DISMISSED** [review]

File: docs/pull_request_template.md (hunk 1 above)
Comment (juliatan) [reply docs/pull_request_template.md]: Yup, will do - thanks!

**Review by selvach — APPROVED** [review]
Nice! Thank you!!

**Review by bionara — APPROVED** [review]
nice one!

**Pull request merged**